docs/further.md

```
@@ -1,6 +1,24 @@
+
+#### Usage Hints
+
+To use this plugin, you can install it into your Snakemake base environment with conda. It will install its dependency, the "jobstep" plugin (which will be used on the cluster nodes). We recommend installing the `snakemake-storage-plugin-fs` (see below) for automatic stage-in and -out procedures.
+
+We recommend running Snakemake on the log in node. Occassionally, HPC administrators are opposed to the idea of having a job sheperd running on the login node, because nobody should calculate there. We therefore provide this table of measurements:
+
+| Workflow | Number of local rules  | Total Runtime (hh:mm:ss) | CPU-Time on the login node [user + system] (s) | Fraction |
+|:------------- |:---------------|:-------------|:-------------|:-------------:|
+| (https://github.com/snakemake-workflows/transcriptome-differential-expression)[Transcriptome DiffExp + Fusion detection]        | 4        | 9:15:43      | 225.15 | 0.68
+
+If you want to contribute a similar statistics, please run `/usr/bin/time -v snakemake ...` on your cluster and contribute your stats as an (https://github.com/snakemake/snakemake-executor-plugin-slurm/issue)[issue to the plugin repo on GitHub].
```

markdownlint-cli2 (0.17.2)

Reversed link syntax
(https://github.com/snakemake/snakemake-executor-plugin-slurm/issue)